tests/ssl-opt.sh

```diff
@@ -1864,7 +1869,7 @@ requires_config_enabled MBEDTLS_RSA_C
 requires_config_enabled MBEDTLS_SHA256_C
 run_test    "TLS-ECDHE-RSA Opaque key for server authentication" \
```

Comment:
Minor, pre-existing (see the other comment): suggest "Opaque key for server authentication: TLS-ECHDE-RSA" instead.